src/style/pinny.scss

```diff
@@ -102,3 +105,21 @@
 [hidden] {
     display: none !important;
 }
+
+
+// Shade
+// -----
+//
+// Let shadow fill 100% of viewport background after rotation
+// on Android 4.3 (HTC, Samsung Galazy S3).
+//
+// Use !important to overwrite inline position styles
+
+.shade {
```

Comment:
So if this is a fix that is specific to shade, we should fix it in shade rather than pinny. In terms of adding CSS to fix this, it would be better to do via javascript in the `orientationchange` event. We try to have a minimal CSS footprint in general for plugins so users don't have any unnecessary external dependencies.

Thanks!